package.json

```diff
@@ -6,19 +6,20 @@
     "url": "https://github.com/momentum-design/momentum-ui.git"
   },
   "scripts": {
-    "bootstrap": "yarn build:tokens && lerna bootstrap && lerna link && cd ./web-components && yarn",
+    "bootstrap": "lerna bootstrap && lerna link",
```

Comment:
I think you need to add the `yarn build:tokens` command back to this

Reply:
@pauljeter  Yeah I caught that, back in now